pilot/test/discovery/discovery.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package discovery
+
+import (
+	"time"
+
+	"github.com/golang/protobuf/ptypes"
+	proxyconfig "istio.io/api/proxy/v1/config"
```

Comment:
run `goimports` - there should be a break between third party deps and istio.io deps. Generally, there should be three sets of imports: golang library imports (then a blank line), all third-party imports (then a blank line), then project-internal (i.e. istio.io) imports.